exporter/views.py

```diff
@@ -20,22 +20,28 @@ def download_export(request):
 
     # Set Content-Encoding to skip GZipMiddleware.
     # https://docs.djangoproject.com/en/3.2/ref/middleware/#module-django.middleware.gzip
-    headers = {}
-    if suffix in ("jsonl.gz", "csv.tar.gz"):
-        headers = {"Content-Encoding": "gzip"}
+    if suffix == "jsonl.gz":
+        # https://stackoverflow.com/a/67807011/244258
+        headers = {"Content-Encoding": "gzip", "Content-Type": "application/jsonlines+json"}
+        extension = "jsonl"
+    if suffix == "csv.tar.gz":
+        # https://stackoverflow.com/a/57954148/244258
+        headers = {"Content-Encoding": "gzip", "Content-Type": "application/x-tar"}
+        extension = "csv.tar"
```

Comment:
```suggestion
        extension = "csv.tar.gz"
```

Otherwise the file is not found

Reply:
Ah, at the line `Export(job_id, basename=f"{stem}.{extension}")` it needs to be `Export(job_id, basename=f"{stem}.{suffix}")

The `filename` argument later needs be the uncompressed `extension`.